src/Config/ConfigurationLoader.php

```diff
@@ -48,12 +49,34 @@ public function fetchAndStoreConfiguration(?string $flagETag): void
                 syslog(LOG_WARNING, "[Eppo SDK] Empty or invalid response from the configuration server.");
                 return;
             }
-            $fcr = FlagConfigResponse::fromJson($responseData);
+            $fcr = FlagConfigResponse::fromArray($responseData);
             $banditResponse = null;
-            // TODO: Also check current bandit models loaded for optimized bandit loading.
             if (count($fcr->banditReferences) > 0) {
-                $bandits = $this->apiRequestWrapper->getBandits();
-                $banditResponse = new ConfigResponse($bandits->body, date('c'), $bandits->eTag);
+                $canReuseBandits = true;
```

Comment:
Log changes live here

Reply:
Not obvious to me - what is the code change in this PR to facilitate not loading bandit models?